noir-projects/aztec-nr/aztec/src/messages/logs/note.nr

```diff
@@ -32,17 +34,19 @@ where
 pub fn private_note_to_message_plaintext<Note>(
     note: Note,
     storage_slot: Field,
-) -> [Field; <Note as Packable>::N + 2]
+    randomness: Field,
+) -> [Field; <Note as Packable>::N + 3]
```

Comment:
kill this magic constant (need to check if Noir's type grammar let's me use a constant here though)